loaders/src/glTF/2.0/glTFLoaderInterfaces.ts

```diff
@@ -217,6 +217,8 @@ export interface ITexture extends GLTF2.ITexture, IArrayItem {
  * Loader interface with additional members.
  */
 export interface ITextureInfo extends GLTF2.ITextureInfo {
+    /** false or undefined if the texture holds color data (true if data are roughness, normal, ...) */
+    isNotColorData?: boolean;

```

Comment:
```suggestion
    isNonColorData?: boolean;
```
I would also drop the `is` part of it, but that's not a big deal.